### ISSUE TITLE
Album disc grouping and sorting is wrong

### ISSUE THREAD

**Issue by KingDuckZ** [strawberrymusicplayer/strawberry#438]
For general technical questions and help with technical issues please use the forum on https://forum.strawberrymusicplayer.org/

**Describe the bug**
*tested with albums from airsonic, I don't know about local albums*
Discs are shown as if they were separate albums (Clementine behaviour was better, imo). This makes the third level grouping by disc pretty useless as each second level entry always contains exactly 1 entry.
Also, for albums with 10 or more discs, entries are not sorted correctly as disc 10 gets placed between 1 and 2 (see screenshot).

**To Reproduce**
1. make an album with 10 discs
2. in strawberry go to the subsonic tab, right click on the list and select visualisation options -> group by -> advanced grouping
3. in the popup window select "album artist" for first level, "year - album" for second level and "disc" for the third
4. click ok and go back to the album list

**Expected behavior**
I should see one icon per album even when the album is made of several discs, then drilling down into the album should give me a list of discs (possibly with the relevant thumbnail) sorted correctly (ie: 10 comes after 9).

**Screenshots:**
![](https://i.imgur.com/1x6Fjti.jpg)

**System Information:**
 - Operating system: 5.4.38-gentoo
 - Strawberry Version: current master ec3d11fb278f796450fc590244a5b9881003b4bc

**Comment by jonaski** [comment]
I actually append the Disc thing to the album title in the code, I think I did this a long time ago for TIdal and just adapted the same thing for Subsonic, because Tidal had like 4 duplicates of some titles I had to separate them somehow. But this was all before I added the Album - Disc grouping so I think we can just remove that stuff now.

**Comment by KingDuckZ** [comment]
Speaking for myself, I'm happy enough with a disc sub-grouping under each album, with albums always appearing only once in the list.
